sandbox/public/index.html

```diff
@@ -37,34 +37,34 @@
 
     <script>
       alloy("configure", {
-        orgId: 23784612874,
-        propertyId: 9999999,
+        orgID: 23784612874,
+        propertyID: 9999999,
         dataSet: "dataset1",
         shouldStoreCollectedData: 1,
         device: "Chrome-Mac",
-        collectionUrl: "https://edgegateway.azurewebsites.net",
+        // collectionUrl: "https://edgegateway.azurewebsites.net",
+        collectionUrl: "http://ex-edge.stable-stage.aam-npe.adobeinternal.net/v1",
```

Comment:
Note that when using the real gateway, there's not much for interesting information coming back yet.